app/scripts/controllers/network/provider-api-tests/shared-tests.js

```diff
@@ -2859,45 +2938,6 @@ export function testsForRpcMethodSupportingBlockParam(
       });
     });
 
-    it.each([null, undefined, '\u003cnil\u003e'])(
```

Comment:
This was migrated below to `earliest` block, and to the `block number -> greater than/equal/less than` blocks, because it depended on those factors.

Reply:
this will make it more difficult to reflect these changes onto https://github.com/MetaMask/core/pull/1087.

Please limit the diff on these tests to changes that are absolutely necessary.

Reply:
Do you have any suggestions on alternative approaches? This was my attempt at making minimal changes while still covering the behavior of this middleware.